### PR TITLE
Move handle release in background.

### PR DESCRIPTION
**Pull request by dlibenzi**

### CODE REVIEW

File: third_party/xla_client/xla_computation_client.cc [hunk 1]
```diff
@@ -232,23 +235,35 @@ string XlaComputationClient::GetEffectiveDevice(const string& device) const {
...
 }
 
+void XlaComputationClient::StartHandleReleaser() {
+  std::thread releaser([this]() { HandleReleaser(); });
+  releaser.detach();
+}
+
+void XlaComputationClient::HandleReleaser() {
+  auto period = std::chrono::milliseconds(500);
+  for (;;) {
```
Comment (asuhan) [inline third_party/xla_client/xla_computation_client.cc]: Don't we prefer `while(true)`?

Comment (dlibenzi) [reply third_party/xla_client/xla_computation_client.cc]: Is it?
I have not found anything in the guide or the ToW, but I do not have a particular attachment with nay of the two styles :)

Comment (asuhan) [reply third_party/xla_client/xla_computation_client.cc]: I'd go with `while (true)`.

Comment (dlibenzi) [reply third_party/xla_client/xla_computation_client.cc]: Alright

**Pull request merged**